src/bootstrap/config.rs

```diff
@@ -375,6 +377,7 @@ impl Config {
         config.rust_codegen_backends = vec![INTERNER.intern_str("llvm")];
         config.rust_codegen_backends_dir = "codegen-backends".to_owned();
         config.deny_warnings = true;
+        config.missing_tools = false;
```

Comment:
`config.missing_tools` is never read from `toml.dist.missing_tools`. 

For reference, please search for "src_tarball" in this file to see how the value is propagated from `toml.dist.*` into `config.*`.

Reply:
Sure I will :)

Reply:
I did `set(&mut config.missing_tools, t.missing_tools` when `Some(t) = toml.dist` just like `src_tarball`